README.md

```diff
@@ -61,20 +61,43 @@ make install
 
 ## Building packages
 
-* It is possible to easily build pacakges for Ubuntu and CentOS using
-  the provided `Makefile-build-docker` makefile leveraging the use of
-  docker containers.
+### Building a RPM
+
+The required build dependencies are:
+- rpm-build
+- make
+- rsync
 
 ```sh
-# Building an Ubuntu Xenial deb
-make -f Makefile-build-docker deb
-# Building a CentOS 7 RPM
-make -f Makefile-build-docker rpm
+# Checkout tag to be packaged
+git clone https://github.com/EGI-Foundation/bdii.git
+cd bdii
+git checkout X.X.X
+# Building in a container
+docker run --rm -v $(pwd):/source -it centos:7
+yum install -y rpm-build make rsync
+cd /source && make rpm
 ```
 
+The RPM will be available into the `build/RPMS` directory.
+
+### Building a deb
```

Comment:
could we not just have a `./build-deb.sh` script which gets run:
```
docker run --rm -v $(pwd):/source -it ubuntu:xenial build-deb.sh
```

Reply:
Yes, why not. It could also be possible to edit the Makefile to add a task and to have make doing the container instantiation and build.

Reply:
but for CI/CI it's probably better to not require any dependency but docker